src/landpatterns/BGA/planner.stanza

```diff
@@ -62,74 +87,156 @@ public defmethod pad-generator (x:BGA-PadPlanner, row:Int, column:Int) -> (Dims
 
 
 doc: \<DOC>
-Full Matrix - All Pads in the grid are active
+@title Full-Matrix-Planner
+@desc Full Matrix - All Pads in the grid are active
+@member pad-config Defines the features of each individual BGA pad.
+@member rows Number of rows in the landpattern.
+@member columns Number of columns in the landpattern.
+@member pitch Pitch of the landpattern.
+@member pose Pose of the landpattern.
```

Comment:
`@member` doesn't work here. For `defstruct`, we expect the individual fields to be annotated with `docstrings`. The `@member` is primarily for bundle and module types.